doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-global-settings-panel-dmn-based-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='test-designer-global-settings-panel-dmn-based-proc']
+= Global settings for DMN-based test scenarios
+
+Follow the procedure below to view and edit the global settings of the DMN-based test scenarios.
+
+.Procedure
+. Click `Settings` tab on the right side of the test scenario designer to display the attributes.
+. Configure the following attributes in the `Settings` panel.
+* `Name`: You can change the name of the existing test scenarios by using the by using the `Rename` option from the upper-right toolbar in the designer.
+* `Type`: This attribute specifies it is a DMN-based test scenario and it is read-only.
+* `DMN model`: (Optinal) Enter the DMN model for the test scenario.
+* `DMN name`: This is the name of the DMN model.
+* `DMN namespace`: This is the default namespace for DMN model. 
+. To skip the simulation during the test, select the check box.
```

Comment:
Suggestion - add "Optionally," to step:

s/To skip the simulation during the test, select the check box./ Optionally, to skip the simulation during the test, select the check box.